com.amphi.music.yaml

```diff
@@ -0,0 +1,67 @@
+id: com.amphi.music
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: music
+
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/bin/lib:$LD_LIBRARY_PATH
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.portal.Desktop
```

Comment:
```suggestion
```

Not sure where you read this should be added, but it's an error. It's part of the build failure. I will not outline the others though.